client/src/components/audit-row/resolver.ts

```diff
@@ -0,0 +1,103 @@
+import { Audit } from '../../types';
+
+type Data = { [key: string]: never };
```

Comment:
This typing would eventually cause issue if we need to access information that is nested, for example

```
data.child.child
```

The cleanest way to fix this is to use a data marshal deserializer before passing the payload to the factory. But I would say that is out of the scope of this ticket as we should be focused on developing new features and improving UX for now.

I recommend using the `any` type for now and we can add marshalling later on

Reply:
You should use the `ßwillFixThisTypeLater` helper type so we can track these typing improvements